.github/workflows/adopt-tapir-ci.yml

```diff
@@ -60,6 +60,20 @@ jobs:
         id: build-docker-image-locally
         run: sbt docker:publishLocal
 
+      - name: Run UI build
+        # run this step only once across all configurations
+        if: matrix.scala == 'Scala3' && matrix.json == 'No'
+        id: run-ui-build
+        run: yarn build
+        working-directory: ./ui
```

Comment:
Just put  in package.json
"test": "react-scripts test a --watchAll=false",

The default process use watch and never ends